### ISSUE TITLE
[BUG: Output]

### ISSUE THREAD

**Issue by yusif-askari** [VikParuchuri/marker#1078]
## 📝 Describe the Output Issue

A table gets outputted twice in the markdown. After inspecting the json file found that it labels one table in the pdf both as a table and a forum with almost exact boundaries

## 📄 Input Document

Attach the PDF or input file used.

https://www.klinikum.uni-heidelberg.de/fileadmin/Psychosomatische_Klinik/pdf_Material/PHQ_Komplett_Fragebogen1.pdf

## 📤 Current Output

Paste the Markdown or HTML that Marker generated:

```markdown
<table><thead><tr><th>12</th><th>Wie stark fühlten Sie sich im Verlauf der letzten 4 Wochen Nicht Wenig Stark</th></tr></thead><tbody><tr><td></td><td>durch die folgenden Beschwerden beeinträchtigt? beeinträchtigt beeinträchtigt beeinträchtigt</td></tr><tr><td>a.</td><td>Sorgen über Ihre Gesundheit</td></tr><tr><td>b.</td><td>Sorgen über Ihr Gewicht oder Ihr Aussehen</td></tr><tr><td>c.</td><td>Wenig oder kein sexuelles Verlangen oder Vergnügen beim</td></tr><tr><td>d.</td><td>Schwierigkeiten mit dem Ehepartner, Lebensgefährten,</td></tr><tr><td>e.</td><td>Belastung durch die Versorgung von Kindern, Eltern oder</td></tr><tr><td>f.</td><td>Stress bei der Arbeit oder in der Schule</td></tr><tr><td>g.</td><td>Finanzielle Probleme oder Sorgen</td></tr><tr><td>h.</td><td>Niemanden zu haben, mit dem man Probleme besprechen kann</td></tr><tr><td>i.</td><td>Etwas Schlimmes, das vor kurzem passiert ist</td></tr><tr><td>j.</td><td>Gedanken an schreckliche Ereignisse von früher oder Träume darüber</td></tr><tr><td>13</td><td>Sind Sie im letzten Jahr geschlagen, getreten oder anderweitig NEIN JA</td></tr></tbody></table>

<table><thead><tr><th>12</th><th>Wie stark fühlten Sie sich im Verlauf der letzten 4 Wochen Nicht Wenig Stark</th></tr></thead><tbody><tr><td></td><td>durch die folgenden Beschwerden beeinträchtigt? beeinträchtigt beeinträchtigt beeinträchtigt</td></tr><tr><td>a.</td><td>Sorgen über Ihre Gesundheit</td></tr><tr><td>b.</td><td>Sorgen über Ihr Gewicht oder Ihr Aussehen</td></tr><tr><td>c.</td><td>Wenig oder kein sexuelles Verlangen oder Vergnügen beim</td></tr><tr><td>d.</td><td>Schwierigkeiten mit dem Ehepartner, Lebensgefährten,</td></tr><tr><td>e.</td><td>Belastung durch die Versorgung von Kindern, Eltern oder</td></tr><tr><td>f.</td><td>Stress bei der Arbeit oder in der Schule</td></tr><tr><td>g.</td><td>Finanzielle Probleme oder Sorgen</td></tr><tr><td>h.</td><td>Niemanden zu haben, mit dem man Probleme besprechen kann</td></tr><tr><td>i.</td><td>Etwas Schlimmes, das vor kurzem passiert ist</td></tr><tr><td>j.</td><td>Gedanken an schreckliche Ereignisse von früher oder Träume darüber</td></tr><tr><td>13</td><td>Sind Sie im letzten Jahr geschlagen, getreten oder anderweitig NEIN JA</td></tr></tbody></table>

**14 Was belastet Sie zur Zeit in Ihrem Leben am meisten?**

**NEIN JA**

**15 Nehmen Sie Medikamente gegen Angst, Depressionen oder Stress?**

#### **16 Nur für Frauen: Fragen zum Thema Monatsblutung, Schwangerschaft und Geburt**

a. Wodurch wird Ihre Monatsblutung am besten beschrieben?

<table><thead><tr><th></th><th>Monats- Keine Monats- Monatsblutung ist Keine</th><th>Monatsblutung bei</th></tr></thead><tbody><tr><td></td><td>blutung ist blutung aufgrund unregelmäßig geworden Monatsblutung</td><td>Hormontherapie</td></tr><tr><td></td><td>unverändert von Schwanger- bzw. Dauer, Abstand oder seit mindestens</td><td>(Einnahme von Östroge</td></tr><tr><td></td><td>schaft oder Stärke haben sich einem Jahr</td><td>nen) oder Verhütung</td></tr><tr><td></td><td>kürzlicher Geburt verändert</td><td>durch die Pille</td></tr><tr><td>b.</td><td>Haben Sie in der Woche vor dem Beginn Ihrer Monatsblutung</td><td></td></tr><tr><td>c.</td><td>Wenn „JA“: Verschwinden diese Probleme am Ende Ihrer Monatsblutung wieder?</td><td></td></tr><tr><td>d.</td><td>Haben Sie während der letzten 6 Monate ein Kind geboren?</td><td></td></tr><tr><td>e.</td><td>Hatten Sie während der letzten 6 Monate eine Fehlgeburt?</td><td></td></tr><tr><td>f.</td><td>Haben Sie Schwierigkeiten, schwanger zu werden?</td><td></td></tr><tr><td></td><td>NEIN</td><td>JA</td></tr></tbody></table>

Dieser Fragebogen ist ein wichtiges Hilfsmittel, um Ihnen die bestmögliche Behandlung zukommen zu lassen. Ihre Antworten können Ihrem Arzt helfen, Ihre Beschwerden besser zu verstehen. Bitte beantworten Sie jede Frage, so gut Sie können. Überspringen Sie Fragen bitte nur, wenn Sie dazu aufgefordert werden.

**Name: Alter: Geschlecht:** weiblich männlich **Datum:**

<table><thead><tr><th>1</th><th>Wie stark fühlten Sie sich im Verlauf der letzten 4 Wochen Nicht Wenig Stark</th></tr></thead><tbody><tr><td></td><td>durch die folgenden Beschwerden beeinträchtigt? beeinträchtigt beeinträchtigt beeinträchtigt</td></tr><tr><td>a.</td><td>Bauchschmerzen</td></tr><tr><td>b.</td><td>Rückenschmerzen</td></tr><tr><td>c.</td><td>Schmerzen in Armen, Beinen oder Gelenken (Knie, Hüften usw.)</td></tr><tr><td>d.</td><td>Menstruationsschmerzen oder andere Probleme mit der Menstruation</td></tr><tr><td>e.</td><td>Schmerzen oder Probleme beim Geschlechtsverkehr</td></tr><tr><td>f.</td><td>Kopfschmerzen</td></tr><tr><td>g.</td><td>Schmerzen im Brustbereich</td></tr><tr><td>h.</td><td>Schwindel</td></tr><tr><td>i.</td><td>Ohnmachtsanfälle</td></tr><tr><td>j.</td><td>Herzklopfen oder Herzrasen</td></tr><tr><td>k.</td><td>Kurzatmigkeit</td></tr><tr><td>l.</td><td>Verstopfung, nervöser Darm oder Durchfall</td></tr><tr><td>m.</td><td>Übelkeit, Blähungen oder Verdauungsbeschwerden</td></tr></tbody></table>

<table><thead><tr><th>2</th><th>Wie oft fühlten Sie sich im Verlauf der letzten 2 Wochen Über- An An mehr Bei</th></tr></thead><tbody><tr><td></td><td>durch die folgenden Beschwerden beeinträchtigt? haupt einzelnen als der Hälfte nahe</td></tr><tr><td></td><td>nicht Tagen der Tage jeden Tag</td></tr><tr><td>a.</td><td>Wenig Interesse oder Freude an Ihren Tätigkeiten</td></tr><tr><td>b.</td><td>Niedergeschlagenheit, Schwermut oder Hoffnungslosigkeit</td></tr><tr><td>c.</td><td>Schwierigkeiten, ein- oder durchzuschlafen, oder vermehrter Schlaf</td></tr><tr><td>d.</td><td>Müdigkeit oder Gefühl, keine Energie zu haben</td></tr><tr><td>e.</td><td>Verminderter Appetit oder übermäßiges Bedürfnis zu essen</td></tr><tr><td>f.</td><td>Schlechte Meinung von sich selbst; Gefühl, ein Versager zu sein</td></tr><tr><td>g.</td><td>Schwierigkeiten, sich auf etwas zu konzentrieren, z. B. beim</td></tr><tr><td>h.</td><td>Waren Ihre Bewegungen oder Ihre Sprache so verlangsamt,</td></tr><tr><td>i.</td><td>Gedanken, dass Sie lieber tot wären oder sich Leid zufügen möchten</td></tr></tbody></table>

**6 Fragen zum Thema "Essen" NEIN JA**

a. Haben Sie öfter das Gefühl, Sie könnten nicht kontrollieren,

wie viel und was Sie essen?

b. Essen Sie öfter – in einem Zeitraum von 2 Stunden – Mengen, die andere Leute als ungewöhnlich groß bezeichnen würden?

**Wenn "NEIN" bei a oder b, gehen Sie bitte zu Frage 9.** c. Ist dies während der letzten 3 Monate im Durchschnitt mindestens zweimal in der Woche vorgekommen?

**7 Haben Sie während der letzten 3 Monate öfter eine oder mehrere der folgenden Maßnahmen unternommen, um eine**

**Gewichtszunahme zu vermeiden? NEIN JA**

a. Sich selbst zum Erbrechen gebracht?

b. Mehr als die doppelte empfohlene Dosis eines Abführmittels

eingenommen?

c. Gefastet, d. h. mindestens 24 Stunden lang nichts gegessen?

d. Mehr als eine Stunde Sport getrieben mit dem ausschließlichen Ziel, nicht zuzunehmen, wenn Sie wie oben beschrieben (6a oder 6b)

gegessen haben?

**8 Wenn Sie bei einer oder mehrerer dieser Maßnahmen,**

**die eine Gewichtszunahme vermeiden sollen, "JA" NEIN JA**

**angekreuzt haben, kam eine davon im Durchschnitt**

**mindestens zweimal in der Woche vor?**

**NEIN JA**

**9 Trinken Sie manchmal Alkohol (einschließlich Bier oder Wein)?**

**Wenn "NEIN", gehen Sie bitte weiter zu Frage 11.**

**10 Ist bei Ihnen im Laufe der letzten 6 Monate mehr als einmal eine**

**der folgenden Situationen eingetreten? NEIN JA**

a. Sie haben Alkohol getrunken, obwohl Ihnen ein Arzt angeraten hat, aus gesundheitlichen Gründen mit dem Trinken aufzuhören? b. Sie haben bei der Arbeit, in der Schule, bei der Versorgung der Kinder oder bei der Wahrnehmung anderer Verpflichtungen Alkohol getrunken, waren angetrunken oder "verkatert"? c. Sie sind der Arbeit, der Schule oder anderen Verpflichtungen fern geblieben oder sind zu spät gekommen, weil Sie getrunken hatten

oder "verkatert" waren?

d. Sie hatten Schwierigkeiten, mit anderen auszukommen, weil Sie

getrunken hatten?

e. Sie sind Auto gefahren, nachdem Sie mehrere Gläser Alkohol

bzw. zu viel getrunken hatten?

**11 Wenn eines oder mehrere der bisher in diesem Fragebogen beschriebenen Probleme bei Ihnen vorliegen, geben Sie bitte an, wie sehr diese Probleme es Ihnen erschwert haben, Ihre Arbeit zu tun, Ihren Haushalt zu regeln oder mit anderen Menschen**

**3 Fragen zum Thema "Angst" NEIN JA**

a. Hatten Sie in den letzten 4 Wochen eine Angstattacke (plötzliches Gefühl der Furcht oder Panik)? **Wenn "NEIN", gehen Sie bitte weiter zu Frage 5.** b. Ist dies bereits früher einmal vorgekommen?

c. Treten manche dieser Anfälle völlig unerwartet auf – d. h. in Situationen, in denen Sie nicht damit rechnen, dass Sie

angespannt oder beunruhigt reagieren?

d. Empfinden Sie diese Anfälle als stark beeinträchtigend, und/oder haben Sie Angst vor erneuten Anfällen?

**4 Denken Sie bitte an Ihren letzten schlimmen Angstanfall. NEIN JA**

a. Bekamen Sie schlecht Luft?

b. Hatten Sie Herzrasen, Herzklopfen oder unregelmäßigen Herzschlag?

c. Hatten Sie Schmerzen oder ein Druckgefühl in der Brust?

d. Haben Sie geschwitzt?

e. Hatten Sie das Gefühl zu ersticken?

f. Hatten Sie Hitzewallungen oder Kälteschauer?

g. Wurde Ihnen übel, hatten Sie Magenbeschwerden oder das Gefühl,

Sie würden Durchfall bekommen?

h. Fühlten Sie sich schwindelig, unsicher, benommen oder einer

Ohnmacht nahe?

i. Spürten Sie ein Kribbeln oder hatten Sie ein Taubheitsgefühl in

Teilen Ihres Körpers? j. Zitterten oder bebten Sie?

k. Hatten Sie Angst, Sie würden sterben?

**5 Wie oft fühlten Sie sich im Verlauf der letzten 4 Wochen Überhaupt An einzelnen An mehr als der durch die folgenden Beschwerden beeinträchtigt? nicht Tagen Hälfte der Tage**

a. Nervosität, Ängstlichkeit, Anspannung oder übermäßige

Besorgnis

**Wenn "Überhaupt nicht", gehen Sie bitte weiter zu Frage 6.**

b. Gefühle der Unruhe, sodass Stillsitzen schwer fällt

c. Leichte Ermüdbarkeit

d. Muskelverspannungen, Muskelschmerzen e. Schwierigkeiten beim Ein- oder Durchschlafen

f. Schwierigkeiten, sich auf etwas zu konzentrieren, z. B. beim

Lesen oder beim Fernsehen

g. Leichte Reizbarkeit, Überempfindlichkeit
```

## ✅ Expected Output

Want for only a single table to be exported

## ⚙️ Environment

Please fill in all relevant details:

* **Marker version**: 2.0.0 (marker-pdf)
* **Surya version**: 0.22.1 (surya-ocr)
* **Python version**: 3.12.11 (CPython)
* **PyTorch version**: 2.13.0
* **Transformers version**: 5.14.1
* **Operating System**: macOS 26.5.2 (Darwin 25.5.0), arm64 / Apple Silicon

## 📟 Command or Code Used

Paste the **exact bash command** or **Python code** you used to run Marker:

<details>
<summary>Click to expand</summary>

```python
import os

os.environ["TORCH_DEVICE"] = "cpu"
os.environ.setdefault("OMP_NUM_THREADS", "8")

import torch
from marker.converters.pdf import PdfConverter
from marker.models import create_model_dict
from marker.output import text_from_rendered

torch.set_num_threads(8)

converter = PdfConverter(
    artifact_dict=create_model_dict(device="cpu"),
    config={
        "disable_ocr": True,
        "output_format": "markdown",
        "html_tables_in_markdown": True,
        "pdftext_workers": 8,
    },
)
markdown, _, images = text_from_rendered(converter("phq_questionnaire.pdf"))
print(len(markdown), "chars,", len(images), "images")
```

</details>

**Comment by zeweihan** [comment]
Strong catch — this dual-emission (same region labeled both `table` and `form` with near-identical bbox, then rendered twice) is the **inverse failure of silent-loss**, and for regulated corpora it's just as costly.

A few angles from the legal-document side:

**1. Duplication corrupts RAG retrieval via frequency bias.**
When a fee-table row or a termination clause gets ingested twice, the duplicated chunk scores higher in retrieval (term-frequency / embedding-cluster reinforcement). Downstream, an LLM reading "two consistent mentions" treats the clause as *reinforced* and authoritative — when in reality it's one clause counted twice. The model becomes *more* confident in a wrong direction, with no error signal anywhere.

**2. It's structurally the same class as cross-column reading-order splicing.**
The layout model assigns two conflicting `structural_role`s to overlapping regions, but there's no conflict-resolution / overlap-dedup step before emission. The fix surface is the same: detect region overlap (IoU) *before* serializing, not after.

**3. Form-like legal pages are the worst case for this.**
Compliance checklists, exhibit cover sheets, court filings, and signature/witness blocks are **simultaneously** tables (row/column structure) and forms (key-value fields). On these page types dual emission is close to guaranteed, not an edge case — which means the `PHQ_Komplett` questionnaire here is representative of a large family of regulated documents, not an exotic input.

**4. "Plausible-but-redundant" is harder to catch than "obviously-broken."**
The output is valid markdown that looks reasonable — just doubled. A human skimming wouldn't necessarily flag it, and no pipeline stage raises an error. This is the same "returns exit code 0" silent-failure pattern that makes layout/extraction bugs so dangerous in document-AI.

**Suggestion:** when two emitted elements share an IoU above a threshold (say >0.7), keep the higher-confidence label and drop the duplicate — but emit a `deduped_overlaps` count (and ideally the surviving label per overlap) in the page-level metadata. That turns a silent merge decision into an *informed* one, so downstream consumers (and humans auditing the extraction) know a dedup happened rather than trusting the clean-looking output.

Happy to provide a couple of redacted contract pages where a clause-table and a signature form-block overlap, if a fixture would help pin down the boundary case.
